<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab1DevControl</class>
 <widget class="QWidget" name="Tab1DevControl">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,6,2">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="pPBtimerStart">
       <property name="text">
        <string>TimerStart</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pCBtimerValue">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>150</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBtquit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDial" name="pDialLed">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchTarget">
          <double>3.700000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="pLcdNumberLed">
         <property name="digitCount">
          <number>2</number>
         </property>
         <property name="mode">
          <enum>QLCDNumber::Mode::Hex</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="pProgressBarLed">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pDialLed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pLcdNumberLed</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pDialLed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pProgressBarLed</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
